tests/test_cross_compile.py

```diff
@@ -755,6 +763,122 @@
 """  # noqa
 
 
+python_recipe_noinspectlinkages = """\
+{% set version = "1.19.0" %}
+
+package:
+  name: numpy
+  version: {{ version }}
+
+source:
+  url: https://github.com/numpy/numpy/releases/download/v{{ version }}/numpy-{{ version }}.tar.gz
+  sha256: 153cf8b0176e57a611931981acfe093d2f7fef623b48f91176efa199798a6b90
+
+build:
+  number: 0
+  skip: true  # [py27]
+  entry_points:
+    - f2py = numpy.f2py.f2py2e:main  # [win]
+
+requirements:
+  host:
+    - python
+    - pip
+    - cython
+    - libblas
+    - libcblas
+    - liblapack
+  run:
+    - python
+
+test:
+  requires:
+    - pytest
+    - hypothesis
+  commands:
+    - f2py -h
+    - export OPENBLAS_NUM_THREADS=1  # [unix]
+    - set OPENBLAS_NUM_THREADS=1  # [win]
+    - conda inspect linkages  # comment
```

Comment:
We should probably add a conda inspect object here too.

Reply:
done!